test_data/transaction_write_load.js

```diff
@@ -0,0 +1,30 @@
+/* global ARGUMENTS, print */
+
+const _ = require('lodash');
+const arangodb = require('@arangodb');
+const db = arangodb.db;
+const internal = require("internal");
+
+let args = _.clone(ARGUMENTS);
+
+let options = internal.parseArgv(args, 0);
+
+print('nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn');
+let c = db._create(`transaction_${options.count}`, {numberOfShards: 2});
+
+let count = 0;
+while (count < 100000) {
```

Comment:
This will run for at least 100.000 * 0.1 seconds = 10.000 seconds = ~3 hours. Is that intentional?

Reply:
so far the intention is that it just won't be finished during the tests.